### PR TITLE
Fix small typos

### PR DESCRIPTION
**Pull request by merkata**
Small typos noticed and fixed in:

- configure-upgrade-etcd.md

- reconfigure-kubelet.md

Signed-off-by: Mariyan Dimitrov <mariyan.dimitrov@gmail.com>

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-18886--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/reconfigure-kubelet/

https://deploy-preview-18886--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/configure-upgrade-etcd/

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ etcd2 and etcd3 is as follows:
                       message `etcd2 is no longer a supported storage backend`
 
 Before upgrading a v1.12.x kube-apiserver using `--storage-backend=etcd2` to
-v1.13.x, etcd v2 data MUST by migrated to the v3 storage backend, and
+v1.13.x, etcd v2 data MUST be migrated to the v3 storage backend, and
 kube-apiserver invocations changed to use `--storage-backend=etcd3`.
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: Hello @merkata . Possible edits of lines 217 - 219:

Before upgrading a  v1.12.x kube-apiserver (using `--storage-backend=etcd2`), etcd2 data must be migrated to the v3 storage backend and kube-apiserver invocations must be changed to use `--storage-backend=etcd3`.

Comment (merkata) [reply content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: @kbhawkey thanks for the suggestion, I've rephrased the sentence.

**Pull request merged**